pyanaconda/installation.py

```diff
@@ -71,6 +71,70 @@ def _writeKS(ksdata):
         f.write(str(ksdata))
 
 
+def _copy_logs(*args):
+    """Copy installation logs to the target system.
+
+    The code is based on the 99-copy-logs.ks post script which is replaced by this.
+    """
+    import os.path
+    import shutil
+    import glob
+
+    def copy_file_if_exists(src, dest):
+        if os.path.exists(src):
+            shutil.copyfile(src, dest)
+
+    log.info("Copying logs from the installation environment...")
+
+    if conf.anaconda.save_logs:
+        util.mkdirChain(conf.target.system_root + "/var/log/anaconda/")
+        # copy log files from the list
+        LOG_FILES_TO_COPY = [
+            "anaconda.log", "syslog", "X.log", "program.log", "packaging.log", "storage.log",
+            "ifcfg.log", "lvm.log", "dnf.librepo.log", "hawkey.log", "dbus.log"
+        ]
+        for logfile in LOG_FILES_TO_COPY:
+            src_path = "/tmp/" + logfile
+            dest_path = conf.target.system_root + "/var/log/anaconda/" + logfile
+            copy_file_if_exists(src_path, dest_path)
```

Comment:
Ideally, the task could have functions like `_copy_file` and `_copy_tree` and the `run` method would just contain a simple series of these calls. Also, it would be great to log what we are copying in these functions.